includes/templates/reader-activation/login-form.php

```diff
@@ -15,9 +15,11 @@
 }
 
 \do_action( 'woocommerce_before_customer_login_form' );
+
+$in_modal = false;
 ?>
 <div class="newspack-ui newspack-reader-auth__inline-wrapper">
-	<?php Reader_Activation::render_auth_form(); ?>
+	<?php Reader_Activation::render_auth_form( $in_modal ); ?>
```

Comment:
Nonblocking nit: It looks like we aren't really modifying the `$in_modal` variable. Can we just call `render_auth_form` with `false`?

```suggestion
?>
<div class="newspack-ui newspack-reader-auth__inline-wrapper">
	<?php Reader_Activation::render_auth_form( false ); ?>
```

Reply:
I was overthinking this 😆 Fixed in [f70a4b9](https://github.com/Automattic/newspack-plugin/pull/3301/commits/f70a4b9f1ae2180e753799558ba0e34810950907) -- thanks!